MiniApp/Classes/JavascriptBridge/bridge.js

```diff
@@ -1,13 +1,14 @@
-var messageQueue = []
-window.MiniAppBridge = {};
+var MiniAppBridge = {};
+MiniAppBridge.messageQueue = [];
 var uniqueId = Math.random();
+
 var isPlatform = {
-    Android: function() {
-        return navigator.userAgent.match(/Android/i);
-    },
-    iOS: function() {
-        return navigator.userAgent.match(/iPhone|iPad|iPod/i);
-    }
+  Android: function() {
```

Comment:
do we ever call isPlatform.Android() somewhere ?

Reply:
Not from iOS but the same generic bridging script has been used by Android too right?